lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -1758,6 +1759,11 @@ DWARFASTParserClang::ParseStructureLikeDIE(const SymbolContext &sc,
   }
 
   if (attrs.is_forward_declaration) {
+    Progress progress(llvm::formatv(
+        "Parsing forward declaration {0}: {1}",
```

Comment:
It should either be "Resolving forward declaration" or "Parsing type". Otherwise it sounds like we take a long time to load a forward declaration itself.

Reply:
done